### DISCUSSION TITLE
Sync Deepgram timings with preexisting script/transcript?

### DISCUSSION THREAD

**Discussion by jahamed** [deepgram/community#282] [General help] [answered]
I have a script that I use TTS software to turn into an audio file. I'm using that audio file with Deepgram to get word level timings to make time accurate subtitles.

But is it possible to include punctuation & formatting from my original script with the Deepgram timings?

Example:
My Script : 'He moved (silently) through the night.'
DG: 
```
{
                "word": "moved",
                "start": 4.87756,
                "end": 5.15742,
                "confidence": 0.99902344,
                "punctuated_word": "moved"
              },
              {
                "word": "silently",
                "start": 5.15742,
                "end": 5.65742,
                "confidence": 1,
                "punctuated_word": "silently"
              },
```

Is there a way to add the () around silently?

Does Deepgram have an option to upload the original transcript? Or is there some type of algorithmic approach I can take here? Thanks!

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @jahamed, Deepgram does not support accepting a user-provided transcript as input. Its transcription should closely match your original script. Perhaps you could use a version of your script with stripped punctuation to align it with Deepgram's output? This would need to be your own post-processing though.
